cgmanifest.json

```diff
@@ -4140,8 +4140,8 @@
         "type": "other",
         "other": {
           "name": "gdisk",
-          "version": "1.0.9",
-          "downloadUrl": "https://downloads.sourceforge.net/gptfdisk/gptfdisk-1.0.9.tar.gz"
+          "version": "1.0.10",
+          "downloadUrl": "https://downloads.sourceforge.net/gptfdisk/gptfdisk-1.0.10.tar.gz"
```

Comment:
There is a PR check relating to this is failing. Please change the name from gdisk to gptfdisk. Take a look at the PR check failure for more details.

Reply:
updated the naming as you suggested.